docs/02-about-us/guilds.md

```diff
@@ -0,0 +1,44 @@
+# Agile Guilds
+
+As [described in our continous learning page](../04-how-we-work/continuous-learning.md), CivicActions leverages the 
+concept of an Agile Guild to harness organizational feedback from team members and project retrospectives.
+
+## Governance of a Guild
+
+* Meet regularly and schedule recurring meetings with screen sharing on open, shared calendars
+* Keep up-to-date meeting notes on Trello cards, e-mail lists, and/or Slack channel
+* Announce meetings at least one hour before hand to encourage participation
+* Participate in project retrospectives as needed, capture feedback into Guild
+* Use the established forums as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
+* Update evolved guild operations in the documentation handbook
+
+## Creating a Guild
+* Establish Trello board (or organizational tool), e-mail list, and slack channel
+* Identify base membership
```

Comment:
I think we should consider a facilitator that guarantees attendance and performs routine tasks (e.g. take notes, post reminders to Slack, organize the board, etc)

Reply:
I don't consider this a cosmetic thing, @fmizzell. Guilds and Practice Areas are radically different concepts, deceptively similar—and this isn't a discussion to have in documentation. The documentation should reflect decisions made, and it sounds like decisions haven't been made.